modules/virt-overriding-default-fs-overhead-value.adoc

```diff
@@ -33,16 +33,16 @@ spec:
       storageClass:
         <storage_class_name>: "<new_value_for_this_storage_class>" <2>
 ----
-<1> The file system overhead percentage that CDI uses across the cluster. For example, `global: "0.07"` reserves 7% of the PVC for file system overhead.
+<1> The default file system overhead percentage used for all storage classes. For example, `global: "0.07"` reserves 7% of the PVC for file system overhead.
 <2> The file system overhead percentage for the specified storage class. For example, `mystorageclass: "0.04"` changes the default overhead value for PVCs in the `mystorageclass` storage class to 4%.
```

Comment:
I think it's still not clear from the wording that the per-storage class value overrides the global value.
(otherwise looks good to me)